src/phoenix/server/api/routers/auth.py

```diff
@@ -187,4 +207,96 @@ async def refresh_tokens(request: Request) -> Response:
     return response
 
 
+@router.post("/password-reset-email")
+async def initiate_password_reset(request: Request) -> Response:
+    data = await request.json()
+    if not (email := data.get("email")):
+        raise MISSING_EMAIL
```

Comment:
Non-blocking: Might be simpler and more declarative to create a payload dataclass and let fastapi do the validation.